.prettierrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "singleQuote": true,
+  "semi": true,
+  "trailingComma": "all",
+  "printWidth": 80,
+  "tabWidth": 2,
+  "arrowParens": "always",
```

Comment:
Blocking: arrrowParens should be `avoid` because we need to omit the parantheses around a sole arrow function parameter when possible